runtime/onert/api/src/nnfw_api.cc

```diff
@@ -494,6 +494,8 @@ NNFW_STATUS nnfw_train_export_circle(nnfw_session *session, const char *)
   return NNFW_STATUS_ERROR;
 }
 
+#endif // ONERT_TRAIN
```

Comment:
Bug fix: quantization API should be outside of `#ifdef ONERT_TRAIN` ~ `#endif` block.